src/java.net.http/share/classes/java/net/http/HttpResponse.java

```diff
@@ -748,6 +749,34 @@ public static <T> BodyHandler<T> buffering(BodyHandler<T> downstreamHandler,
                      .buffering(downstreamHandler.apply(responseInfo),
                                 bufferSize);
          }
+
+        /**
+         * {@return a {@code BodyHandler} limiting the number of bytes consumed
+         * and passed to the given downstream {@code BodyHandler}}
+         * <p>
+         * If the number of bytes received exceeds the maximum number of bytes
+         * desired as indicated by the given {@code capacity},
```

Comment:
Should we consider simplifying this sentence to "If the number of bytes received exceeds the given {@code capacity}, ..."?

Reply:
Good suggestion.

Reply:
Fixed in 23f79a20476f784058801454a28d37f37df92982.